CHANGES_NEXT_RELEASE

```diff
@@ -25,6 +25,7 @@ Add:  URI parameter 'q' for GET /v2/entities (Issue #967)
 Add:  PUT /v2/entities/<entityId>/attrs/<attrName>/value (Issue #999)
 Add:  DELETE /v2/entities/<entityId>/attrs/<attrName> (Issue #993)
 Add:  PUT /v2/entities/<entityId>/attrs/<attrName> (Issue #991)
-Add:  GET /v2/types/<type> (Issue #1028)
+Add:  GET /v2/type/<type> (Issue #1028)
```

Comment:
It should be "/v2/types" in plural, as "/v2/entities", "/v2/registrations", "/v2/subscriptions". All the operations in the NGSIv2 use  plural in the URL (I think is a common idiom in REST).

I think the .apib has a typo in the retrieve entity type operation. It should be also fixed.